### PR TITLE
Convert userPasswordPage & UserImagePage to react

### PR DESCRIPTION
**Pull request by grafixeyehero**
Depend on  Convert userParentalControlPage to react #3072

### CODE REVIEW

**Comment by thornbill** [comment]
@jellyfin-bot rebase

**Comment by thornbill** [comment]
@grafixeyehero It looks like the bot wasn't able to rebase this PR. Would you have some time to update it? We have made some TypeScript changes recently that might impact this. Thanks!

File: src/components/pages/UserImagePage.tsx [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+import ButtonElement from '../dashboard/users/ButtonElement';
+import UserPasswordForm from '../dashboard/users/UserPasswordForm';
+import loading from '../loading/loading';
+import toast from '../toast/toast';
+
+type IProps = {
+    userId: string;
+}
+
+const UserImagePage: FunctionComponent<IProps> = ({userId}: IProps) => {
```
Comment (thornbill) [inline src/components/pages/UserImagePage.tsx:17]: `UserProfilePage` seems like it would be a more accurate name for this page.

**Review by thornbill — CHANGES_REQUESTED** [review]
Seems to work well! Just a few comments. :slightly_smiling_face:

File: src/components/pages/UserImagePage.tsx (hunk 1 above)
Comment (grafixeyehero) [reply src/components/pages/UserImagePage.tsx]: https://github.com/jellyfin/jellyfin-web/pull/3073/commits/7af36ab52b4e6d4e587d868c36d9fd10b3c9d48e

File: src/components/dashboard/users/UserPasswordForm.tsx [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+                    throw new Error('Unexpected null user.Id');
+                }
+
+                if (!user.Configuration) {
+                    throw new Error('Unexpected null user.Configuration');
+                }
+
+                LibraryMenu.setTitle(user.Name);
+
+                const showPasswordSection = true;
```
[the remark this answers is not in the document]
Comment (grafixeyehero) [reply src/components/dashboard/users/UserPasswordForm.tsx]: https://github.com/jellyfin/jellyfin-web/pull/3073/commits/7af36ab52b4e6d4e587d868c36d9fd10b3c9d48e

File: src/apiclient.d.ts [hunk 3]
```diff
@@ -93,7 +93,7 @@ declare module 'jellyfin-apiclient' {
         deleteItemImage(itemId: string, imageType: ImageType, imageIndex?: number): Promise<void>;
         deleteItem(itemId: string): Promise<void>;
         deleteLiveTvRecording(id: string): Promise<void>;
-        deleteUserImage(userId: string, imageType: ImageType, imageIndex?: number): Promise<void>;
+        deleteUserImage(userId: string, imageType: string, imageIndex?: number): Promise<void>;
```
[the remark this answers is not in the document]
Comment (grafixeyehero) [reply src/apiclient.d.ts]: https://github.com/jellyfin/jellyfin-web/pull/3073/commits/7af36ab52b4e6d4e587d868c36d9fd10b3c9d48e

**Review by thornbill — APPROVED** [review]

**Pull request merged**